cw/tvla.py

```diff
@@ -17,7 +17,7 @@
 import yaml
 from chipwhisperer.analyzer import aes_funcs
 from joblib import Parallel, delayed
-from scipy.stats import ttest_ind_from_stats
+from util_tvla.ttest import ttest_hist_xy
```

Comment:
I don't have a strong opinion about this but I think it would make sense to actually move this `ttest.py` out of the `cw` folder into the `util` folder at the top level of the repo. Because it's not CW specific. Also it doesn't need to go into a tvla-specific utility folder. WDYT?

Reply:
I agree. I've moved 'ttest.py' to `util`